lib/resources/example_json/dstu2_examples_medication_administration.rb

```diff
@@ -0,0 +1,429 @@
+module Cerner
+  module Resources
+
+    DSTU2_MEDICATION_ADMINISTRATION_BUNDLE ||= {
+      "resourceType": "Bundle",
+      "id": "a9117d85-042f-4dda-9e38-dce0286c57a4",
+      "type": "searchset",
+      "link": [
+        {
+          "relation": "self",
+          "url": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationAdministration?patient=4642007"
```

Comment:
Should be using the urls: fhir-open.sandboxcerner.com

Reply:
Changed: https://github.com/cerner/fhir.cerner.com/pull/169/commits/a18a46531f69088dec1ee6993ef849fb497c772f#diff-e8b526c8fb4dde5a442627faec99bc0f